src/lib.rs

```diff
@@ -12,6 +12,56 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+//! Cross platform GFX example application framework.
+//! 
+//! Supports OpenGL 3.2, OpenGL ES 2.0, WebGL 2, DirectX 11, Vulkan, and Metal. Handles windowing via `winit`.
```

Comment:
There is no Vulkan and Metal support in the old gfx-rs. The existing work has not been completed.